SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -109,8 +110,8 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	void LocalDeploymentClicked();
 	void CloudDeploymentClicked();
 
-	bool IsLocalDeploymentIPEditable() const;
-	bool AreCloudDeploymentPropertiesEditable() const;
+	static bool IsLocalDeploymentIPEditable();
```

Comment:
Is there a standard here where we make member function static if they can be?

If so I guess GetPrimaryDeploymentNameText can be static?

Reply:
Yes, I also noticed a lot of the functions in this class as well as `SSpatialGDKCloudDeploymentConfiguration` can be static, but I felt it wasn't really necessary to change them in this PR and we don't really gain much by doing that.